astrodynamics-fall-2018/astrodynamicsFall2018PDD.tex

```diff
@@ -0,0 +1,372 @@
+  For the roll off motor repair, someone with some motor knowledge would be crucial.
+
+\subsection{Funding Requirements}
+  % Estimate costs that would be needed to meet objectives.
+  Primary funding for these projects will come from the Observatory itself.
+  If extra funds are needed, RIT SPEX Astrodynamics may apply for SG funding due to their newly developed SG club status.
+  \autoref{tab:costEstimate} outlines the estimated costs of the projects.
+  The costs have been overestimated for planning purposes.
+
+  \begin{table}[hb!]
+    % this table is too wide for the two-column format, so we let it expand across both columns
+    % we haven't told LaTeX where to put this so it'll find the best place.
+        \caption{Estimated Budget}
+        \centering
+        \begin{tabular}{ll}
+          % READ THIS!! https://www.inf.ethz.ch/personal/markusp/teaching/guides/guide-tables.pdf
+          \toprule % line on top external edge of table
+          % Separate cells in a row with &, move to the next row with \\
+          Project & Cost \\
+          \midrule % line separating two internal rows
+          Roll Off Motor Repair & \$500 \\
+          Motor Latch Arm & \$100 \\
+          12" Telescope Modifications & \$100 \\
+          % LaTeX doesn't really like multi-line cell contents. Try to keep the text in each cell concise!
+          \bottomrule
+        \end{tabular}
+    \label{tab:costEstimate}
+    \end{table}
+
+\subsection{Faculty Support}
+  % Identify faculty that will be involved (or would need to be involved) to meet objectives.
+  % Note that if a professor is the Principal Investigator (P.I.) for a project, there still needs to be a student as the SPEX Project Champion.
+  Direct support from Dr. Richmond and Dr. Connelly is expected.
+  Access to the facility is highly dependent on one of the above being present.
+  Both faculty are very familiar with the equipment, telescopes, and facility.
+  They also have experience with what is typical for systems at other observatories, which may be beneficial when working on these projects.
+  Dr. Richmond has been modifying equipment at the observatory for decades.
+  Dr. Connelly understands the structure and capabilities of RIT SPEX through being an advisor.
+  Both are great resources for guidance during the completion of the projects.
+  As all professors, they are juggling multiple responsibilities at one time, which can complicate scheduling time for this project.
+  Even so, they understand the needs of the observatory and can also be seen as the primary customers for these projects, so it is important to keep them informed through the engineering process
+
+\subsection{Long-Term Vision}
+\label{sec:vision}
+The long term goal of these projects is to work towards a fully automated dome system.
+This would enable remote observing for the Observatory and enable increased access to more students.
```

Comment:
This is really cool but I don't quite see the stepping stones that will lead down this road from this project as a starting point. Are there ideas for incremental progress? How do these projects lay the foundation?